ios general/ios general/CoreData/OrganizationMO+Custom.h

```diff
@@ -0,0 +1,14 @@
+#import "OrganizationMO+CoreDataClass.h"
+#import "OrganizationMO+CoreDataProperties.h"
+
+@class EmployeeMO;
+
+@interface OrganizationMO (Custom)
+
+@property (nonatomic, readonly) NSArray<EmployeeMO *> *sortedEmployeeArray;
```

Comment:
Array types should never have "array" word in their names. Rename to sortedEmployees